tools/esp32/Config.mk

```diff
@@ -80,5 +81,5 @@ define DOWNLOAD
 		echo "USAGE: make download ESPTOOL_PORT=<port> [ ESPTOOL_BAUD=<baud> ]"; \
 		exit 1; \
 	fi
```

Comment:
Now `ESPTOOL_BINDIR` is required for the DOWNLOAD target.
It is important to add a verification for it.

Reply:
```
	$(Q) if [ -z $(ESPTOOL_BINDIR) ]; then \
		echo "DOWNLOAD error: Missing path to bootloader and partition table binary images."; \
		echo "USAGE: make download ESPTOOL_BINDIR=<dir> ESPTOOL_PORT=<port> [ ESPTOOL_BAUD=<baud> ]"; \
		exit 1; \
	fi

Reply:
Part of this PR makes `ESPTOOL_BINDIR` not required for the download target.  So I can do just `make downalod ESPTOOL_PORT=...` and flash only the NuttX image without the rest.

Reply:
Ah, nice. Now I get why you decided to still keep the concatenated variable hehehe

Reply:
I should update the USAGE line though.